frontend/src/main/scala/bloop/cli/Commands.scala

```diff
@@ -28,7 +28,7 @@ object Commands {
   sealed trait RawCommand extends Command
 
   sealed trait CompilingCommand extends RawCommand {
-    def project: String
+    def project: List[String] // FIXME: name change to "projects"?
```

Comment:
I think it's fine to change the name here so long as we change it in the subclasses. When you do, you will also be changing the name of the command! Something we don't want to definitely do, because that would be a breaking change in our CLI. So let's add an extra `ExtraName("project")` before the `projects` definition so that people can continue using their current commands.
  
You will need to change also the part of the logic in `Cli` where we infer the projects from remaining cli args (right now we only infer one project, but now we should infer multiple, and note that we should not infer them for commands like `run`).

Reply:
Also please consider updating documentation if you change argument name :wink: